docs/UserGuide.md

```diff
@@ -60,7 +61,7 @@ statement_guid,property_id,wikidata_value,meta_wikidata_value,external_value,ext
 
 _**Note**: The data `wikidata_value`, `external_value`, `external_url` should be limited to a length of 1500 characters maximum._
 
-Additionally, each row of the csv file must contain exactly 4 commas (`,`). Optional values can simply be left out.
+All columns must be present; optional values can be left empty.
 
 You can consult this [mismatch file](exampleMismatchFile.csv) for a valid example.
```

Comment:
This file mentioned here needs to be updated to be consistent with the new changes

Reply:
Ok, then I’ll push a commit to #456 to update the file there as well. (Here I was mostly following the list of files changed there.)

Reply:
Thanks. That's a good idea.